src/app/components/register/register.component.ts

```diff
@@ -9,6 +9,11 @@ import { Validator } from 'src/app/shared/validators/validator';
   styleUrls: ['./register.component.scss'],
 })
 export class RegisterComponent implements OnInit {
+  
+  @Input() public signUpChecked!: boolean;
+
+  @Input() public loginChecked!: boolean;
```

Comment:
You dont need this flag here. This component is only for register logic

Reply:
done